web/src/components/forms/RuleForm/RuleFormTestResult/RuleFormTestResult.tsx

```diff
@@ -88,7 +101,7 @@ const RuleFormTestResult: React.FC<RuleFormTestResultProps> = ({ testResult }) =
             {dedupFunction && (
               <React.Fragment>
                 <Box as="dt" color="navyblue-100">
-                  Dedup String
+                  Dedup Message
```

Comment:
```suggestion
                  Dedup String
```

Designs have it wrong, we should keep it as is. Sorry for any confusion

Reply:
Fixed, no worries at all :)